docs/backends/dropbox.rst

```diff
@@ -26,5 +26,13 @@ To use DropBoxStorage set::
       Timeout in seconds for making requests to the API. If ``None``, the client will wait forever.
       The default is ``100`` seconds which is the current default in the official SDK.
 
+``DROPBOX_FILE_OVERWRITE`` (optional)
+      Overwrite an existing file when it has the same name as the file being uploaded.
+      Otherwise, rename it. Default is ``False``
+
+``DROPBOX_WRITE_MODE`` (optional)
```

Comment:
Can you default this to whatever the current default is (not overwrite). Don't want to break backwards compat. Can you also link out to the Dropbox docs for this property.

Reply:
changed to 'add'